geo-specific-codelist/gremlin/nuts-geography.grm

```diff
@@ -0,0 +1,2606 @@
+    addE('usedBy').to(
+      g.V().hasLabel('_code_list').has('listID', 'nuts-geography').has('edition', 'one-off')
+    ).property(single, 'label', 'East Surrey')).next()
+
+
+g.V().hasLabel('_code').has('listID', 'geography').has('value', 'UKJ27').
+  fold().coalesce(unfold(), addV('_code').
+    property(single, 'listID', 'geography').property(single, 'value', 'UKJ27')
+  ).next()
+g.V().hasLabel('_code').has('listID', 'geography').has('value', 'UKJ27').outE().fold().coalesce(unfold(),
+  g.V().hasLabel('_code').has('listID', 'geography').has('value', 'UKJ27').
+    addE('usedBy').to(
+      g.V().hasLabel('_code_list').has('listID', 'nuts-geography').has('edition', 'one-off')
+    ).property(single, 'label', 'West Sussex (South West)')).next()
+
+
+g.V().hasLabel('_code').has('listID', 'geography').has('value', 'UKJ28').
+  fold().coalesce(unfold(), addV('_code').
+    property(single, 'listID', 'geography').property(single, 'value', 'UKJ28')
+  ).next()
+g.V().hasLabel('_code').has('listID', 'geography').has('value', 'UKJ28').outE().fold().coalesce(unfold(),
+  g.V().hasLabel('_code').has('listID', 'geography').has('value', 'UKJ28').
+    addE('usedBy').to(
+      g.V().hasLabel('_code_list').has('listID', 'nuts-geography').has('edition', 'one-off')
+    ).property(single, 'label', 'West Sussex (North East)')).next()
+
+
+g.V().hasLabel('_code').has('listID', 'geography').has('value', 'UKJ31').
+  fold().coalesce(unfold(), addV('_code').
+    property(single, 'listID', 'geography').property(single, 'value', 'UKJ31')
+  ).next()
+g.V().hasLabel('_code').has('listID', 'geography').has('value', 'UKJ31').outE().fold().coalesce(unfold(),
+  g.V().hasLabel('_code').has('listID', 'geography').has('value', 'UKJ31').
+    addE('usedBy').to(
+      g.V().hasLabel('_code_list').has('listID', 'nuts-geography').has('edition', 'one-off')
+    ).property(single, 'label', 'Portsmouth')).next()
+
+
+g.V().hasLabel('_code').has('listID', 'geography').has('value', 'UKJ32').
+  fold().coalesce(unfold(), addV('_code').
+    property(single, 'listID', 'geography').property(single, 'value', 'UKJ32')
+  ).next()
+g.V().hasLabel('_code').has('listID', 'geography').has('value', 'UKJ32').outE().fold().coalesce(unfold(),
+  g.V().hasLabel('_code').has('listID', 'geography').has('value', 'UKJ32').
+    addE('usedBy').to(
+      g.V().hasLabel('_code_list').has('listID', 'nuts-geography').has('edition', 'one-off')
+    ).property(single, 'label', 'Southampton')).next()
+
+
+g.V().hasLabel('_code').has('listID', 'geography').has('value', 'UKJ34').
+  fold().coalesce(unfold(), addV('_code').
+    property(single, 'listID', 'geography').property(single, 'value', 'UKJ34')
+  ).next()
+g.V().hasLabel('_code').has('listID', 'geography').has('value', 'UKJ34').outE().fold().coalesce(unfold(),
+  g.V().hasLabel('_code').has('listID', 'geography').has('value', 'UKJ34').
+    addE('usedBy').to(
+      g.V().hasLabel('_code_
```

Comment:
Make sure this file is also regenerated (see comments for corresponding cypher script)